support-workers/src/main/scala/com/gu/support/workers/encoding/Conversions.scala

```diff
@@ -21,9 +21,9 @@ object Conversions {
   }
 
   implicit class FromOutputStream(val self: ByteArrayOutputStream) {
-    def toClass[T](encrypted: Boolean)(implicit decoder: Decoder[T]): T = {
+    def toClass[T](implicit decoder: Decoder[T]): T = {
       val is = self.toInputStream
-      val str = Encryption.decrypt(Streamable.bytes(is), encrypted)
+      val str = new String(Streamable.bytes(is), utf8)
```

Comment:
Same feedback as above, we can pass in `java.nio.charset.StandardCharsets.UTF_8`.